lib/erlef_web/plugs/events.ex

```diff
@@ -16,7 +16,7 @@ defmodule ErlefWeb.Plug.Events do
 
   defp all_events do
     Event
-    |> Posts.all()
+    |> Repo.all()
```

Comment:
We don't want "all" the events. Only events that have been reviewed/approved. The function would be in a query module for events. Calling the `Repo` directly from outside the data layer will cause issues later on